CoreScriptsRoot/Modules/Settings/Pages/ReportAbuseMenu.lua

```diff
@@ -278,6 +278,34 @@ do
 	PageInstance.Displayed.Event:connect(function()
 		PageInstance:UpdatePlayerDropDown()
 	end)
+
+	function PageInstance:ReportPlayer(player)
+		if player then
+			local setReportPlayerConnection = nil
```

Comment:
Lua will evaluate the statement before creating the local, so you can't use it as an upvalue in your event connection. If you want to disconnect it from inside the callback you have to do this.